lib/testutils/minirunner/minirunner.go

```diff
@@ -56,7 +57,7 @@ func (r MiniRunner) MakeArchive() *lib.Archive {
 }
 
 // NewVU returns a new VU with an incremental ID.
-func (r *MiniRunner) NewVU(out chan<- stats.SampleContainer) (lib.VU, error) {
+func (r *MiniRunner) NewVU(_ int64, out chan<- stats.SampleContainer) (lib.InitializedVU, error) {
```

Comment:
True, though I found it easier/safer to keep using this auto-increment feature to avoid having to generate the values in every test. Plus laziness :)

But sure, I'll change it.

Reply:
You'd still have the auto-incrementing feature, only this time in a single place that handles both the JS and our mock `Runner`s

Reply:
Should be fixed in 9470c23a.